### PR TITLE
feat: Apply Cloudscape's `Board Component` into Summary Page 

### LINKED ISSUE #2170: Refactoring summary page
### Main idea

```Java
### Tasks
- [ ] Let's change summary page to React component.
- [ ] Every panel is movable.
- [ ] Every panel is removable.
- [ ] Every panel is resizable.
- [ ] The height and width of panels is multiple size of grid. (see the image below)
- [ ] In the footer, it can include some link such as upload files, webui app download link, etc.
```

### Alternative ideas

_No response_

### Anything else?

Please ref [aws](https://ap-northeast-2.console.aws.amazon.com/console/home?region=ap-northeast-2) page!

- The height and width of panels is multiple size of grid.

- movable, removable, resizable, and it has own grid size.

JIRA Issue: FR-75

### PR DESCRIPTION
**Pull request by ironAiken2**
### Feature 
The scope of this PR stack is to apply `cloudScape's board-component` into the summary page. I used the `patch-package` library to apply the css because that `CloudScape's board-component` doesn't provide any properties for changing the style. 

- Applied cloudscape-design's `board component`
- Modified `board component`'s UI design by using patch-packages.  
- Supports `Dark Mode`

- Remove comments about `summaryPage` in `App.tsx` and `<backend-ai-summary-view>` components in `backend-ai-webui.ts`
- See the summary page

**Checklist:** (if applicable)

- [x] Mention to the original issue
- [ ] Documentation
- [ ] Minium required manager version
- [ ] Specific setting for review (eg., KB link, endpoint or how to setup)
- [ ] Minimum requirements to check during review
- [ ] Test case(s) to demonstrate the difference of before/after

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -116,6 +116,7 @@
     "@types/estree": "1.0.5",
     "@types/hammerjs": "^2.0.45",
     "@types/jest": "^29.5.12",
+    "@types/react-grid-layout": "^1.3.5",
```
Comment (yomybaby) [inline package.json]: You don't need to add this type for web component parts.

File: src/components/backend-ai-webui.ts [hunk 2]
```diff
@@ -884,13 +884,6 @@ export default class BackendAIWebUI extends connect(store)(LitElement) {
         <div id="loading-drag-area" class="loading-background-drag-area"></div>
       </div>
       <div id="app-page">
-        <backend-ai-summary-view
-          class="page"
-          name="summary"
-          ?active="${this._page === 'summary'}"
-        >
-          <mwc-circular-progress indeterminate></mwc-circular-progress>
-        </backend-ai-summary-view>
```
Comment (yomybaby) [inline src/components/backend-ai-webui.ts]: You also can delete this [part](https://github.com/lablup/backend.ai-webui/blob/065f3e2d88a0ccb4c86b12d62e9d4adf976fab8a/src/backend-ai-app.ts#L90-L93).

**Comment by ironAiken2** [comment]
* **#2343** <a href="https://app.graphite.dev/github/pr/lablup/backend.ai-webui/2343?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @ironAiken2 and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

**Review by agatha197 — CHANGES_REQUESTED** [review]
The bottom is too narrow. Can you add more bottom padding?

**Review by agatha197 — APPROVED** [review]
LGTM

**Review by yomybaby — CHANGES_REQUESTED** [review]
@ironAiken2 please resolve conflicts.

**Comment by ironAiken2** [comment]
> @ironAiken2 please resolve conflicts.

@yomybaby Conflict resolved.

**Review by yomybaby — APPROVED** [review]
LGTM

**Pull request merged**